packages/amplify-util-uibuilder/src/commands/utils/codegenResources.ts

```diff
@@ -203,26 +203,17 @@ export const generateAmplifyUiBuilderUtilFile = (context: $TSContext, { hasForms
  * If models are available, they will be populated in the models field of the returned object.
  * If they're not available, it will return undefined
  */
-export const getAmplifyDataSchema = async (studioClient: AmplifyStudioClient): Promise<GenericDataSchema | undefined> => {
-  if (!studioClient.isGraphQLSupported) {
-    return undefined;
-  }
+export const getAmplifyDataSchema = async (context: $TSContext): Promise<GenericDataSchema | undefined> => {
   try {
-    const meta = stateManager.getMeta();
-    const resourceName = Object.entries(meta[AmplifyCategories.API]).find(
-      ([, value]) => (value as { service: string }).service === AmplifySupportedService.APPSYNC,
-    )?.[0];
-    if (resourceName) {
-      const model = await studioClient.getModels(resourceName);
-      if (model) {
-        const source = model.replace(model.substring(0, model.indexOf(`{`) - 1), ``).replace(/;/g, ``);
-        return getGenericFromDataStore(JSON.parse(source));
-      }
+    const localSchema = await context.amplify.invokePluginMethod(context, 'codegen', undefined, 'getModelIntrospection', [context]);
```

Comment:
Please add `codegen` as a dependency of this package

Reply:
Added `"amplify-codegen": "^3.4.0"` to `package.json`